Krypt/Source/SHA256.swift

```diff
@@ -21,6 +22,13 @@ public struct SHA256 {
     return status == errSecSuccess ? Data(hash) : nil
   }
 
+  /// Calculates SHA256 hash of given data using CryptoKit
+  /// - Parameter data: The data to calculate the hash for
+  @available(iOS 13, *)
+  public static func digestV2(_ data: Data) -> Data? {
```

Comment:
There's no v2, should be part of the `digest(_ data: Data) -> Data?` using either CommonCrypto or CryptoKit depending on iOS version.